web-client/src/views/AdvancedSearch/PractitionerSearchByBarNumber.tsx

```diff
@@ -71,14 +74,16 @@ export const PractitionerSearchByBarNumber = connect(
                   id="practitioner-search-by-bar-number-button"
                   onClick={e => {
                     e.preventDefault();
-                    submitPractitionerBarNumberSearchSequence();
+                    submitPractitionerBarNumberSearchSequence({
```

Comment:
We have the current user off of applicationContext, would it be easier to grab that when it's needed rather than passing it around?